Makefile

```diff
@@ -433,3 +434,10 @@ checklinks:
 	command -v markdown-link-check >/dev/null 2>&1 || { echo >&2 "markdown-link-check not installed. Run 'npm install -g markdown-link-check'"; exit 1; }
 	find . -name \*.md -print0 | xargs -0 -n1 \
 		markdown-link-check -q -c ./.github/workflows/check_links_config.json || true
+
+# error message "failed to sync logger:  sync /dev/stderr: inappropriate ioctl for device"
+# is a known issue but does not affect function. 
+.PHONY: crosslink
+crosslink: install-tools
```

Comment:
we don't require install-tools in other places (unless I forget), just try to be consistent.

Reply:
I believe there is some inconsistency. Multimod requires `install-tools`. But I will remove it to match the majority.